src/pages/console/adminui/access-management.mdx

```diff
@@ -41,3 +37,44 @@ Follow these instructions to add and manage team members and their access to a p
 <Callout warning>
 If a team member logs into Amplify Studio, their login token is valid for 60 minutes, unless explicitly logged out. When you change a team member's permission from *full access* to *manage-only* or when you delete a team member's access, the team member can continue accessing Amplify Studio with their previously granted permissions until their token expires.
 </Callout>
+
+## Understanding how Studio manages user access
+
+<Callout warning>
+  The following resources are all managed by Studio so please **do not modify their settings, as they may affect your login experience**.
```

Comment:
> so please **do not modify their settings

we could maybe say `, ___Manual changes to the resource settings, may effect your login experience `

looks like the formatting does occur correctly as well, could you try `_` or ` : 
![image](https://user-images.githubusercontent.com/87995712/222850895-22bb125c-586b-460f-8496-bc8e1d2391f4.png)